packages/worker/src/worker.ts

```diff
@@ -774,7 +782,7 @@ export class Worker<T extends WorkerSpec = DefaultWorkerSpec> {
               { activation, parentSpan, synthetic }
             ): Promise<{
               state: Workflow | undefined;
-              output: ContextAware<{ completion?: Uint8Array; close: boolean }>;
+              output: ContextAware<{ completion?: Uint8Array; close: boolean; runId: string }>;
```

Comment:
No need to add the runId everywhere, it's available as the GroupedObservable's key: https://rxjs-dev.firebaseapp.com/api/index/interface/GroupedObservable